spec/factories/commodity_factory.rb

```diff
@@ -57,5 +57,21 @@
         attributes_for(:import_trade_summary)
       end
     end
+
+    trait :with_conditionally_prohibitive_measures do
+      import_measures do
+        [
+          attributes_for(:measure, :conditionally_prohibitive),
+        ]
+      end
+    end
+
+    trait :with_prohibitive_measures do
+      import_measures do
+        [
+          attributes_for(:measure, :prohibitive),
```

Comment:
`attributes_for_list(:measure, 1, :prohibitive)` will return an array of 1 if that helps?